cli/src/pcluster/constants.py

```diff
@@ -250,6 +250,9 @@ class Feature(Enum):
     FSX_OPENZFS = "FSx OpenZfs"
     SLURM_DATABASE = "SLURM Database"
     CLUSTER_HEALTH_METRICS = "Cluster Health Metrics"
+    INTEL_HPC_SPECIFICATION_2018 = "Intel HPC Specification 2018"
+    INTEL_ONE_API_BASE_TOOLKIT = "Intel OneAPI Base Toolkit"
+    INTEL_PYTHON = "Intel Python"
 
 
 UNSUPPORTED_FEATURES_MAP = {
```

Comment:
Thinking about these features in ADC: does the installation of this package require internet access at runtime?
If so, we should mark these features as unsupported in ADC.

Reply:
Can we list this as a future improvement. Current, we download from Intel, so require Internet access. In the future, we might host the installers and may not require Internet access

Reply:
So with the current implementation we need to set this feature as unsupported for ADC regions in the CLI